control-plane/subcommand/server-acl-init/command.go

```diff
@@ -313,7 +314,6 @@ func (c *Command) Run(args []string) int {
 		scheme = "https"
 	}
 
-	var updateServerPolicy bool
```

Comment:
This is no longer needed. Previously this would be set when there already exists a kube bootstrap secret to ensure that we update the server policy. Now we call `c.bootstrapServers`, even when the kube secret exists. That function then calls `setServerTokens` that then calls `c.setServerPolicy(consulClient)`. So we don't need this codepath anymore.